octopoes/bits/ask_port_specification/question_schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "$id": "/bit/port-classification",
+  "type": "object",
+  "default": {},
+  "Port Configuration": "Root Schema",
+  "required": [],
+  "properties": {
+    "common": {
+      "type": "object",
+      "default": {},
+      "Common Ports": "Allowed ports",
+      "required": [],
+      "properties": {
+        "udp": {
+          "title": "Comma separated list of allowed UDP ports",
```

Comment:
Why not a JSON list instead of a comma separated string?

Reply:
We thought it would perhaps be easier to fill a comma separated list into the generated form in Rocky

Reply:
We initially had an object/array of ports which themselves where numbers. This ended up being a complex form to render and keep clean dynamically. Doing a CSV instead seems user-friendly enough, especially if we provide the correct input-patterns.

Reply:
Maybe we should (also) provide the option for the user to enter JSON in a simple editor, which is then validated against the JSON Schema?

If on the other hand we insist on having a rendered form, why do we go for JSON Schema instead of a simple questions list with key, label and validation pattern (regex)?

Reply:
JsonSchema allows for more complex queries, and provides us with the ability (and tooling) to create schemas, render them into complex forms, and to validate them both client and server side against the given input.
The output of this step is Json, which in itself is an easy to work with and well understood format.

This current question schema is a first example of the use of the 'procedures' code. Future forms might be user-generated, and as such I'd like to keep that flexibility available.